Task3_Products/routers/categories.py

```diff
@@ -0,0 +1,72 @@
+from typing import Literal, Optional
+
+from fastapi import APIRouter, Depends
+from sqlalchemy.orm import Session
+
+from Task3_Products import crud, schemas, database
+
+
+router = APIRouter()
+
+
+@router.post(
+    "/categories/",
+    response_model=schemas.Category,
+    status_code=201
+)
+async def create_category_endpoint(
+    category_data: schemas.CategoryCreate,
+    db: Session = Depends(database.get_db)
+) -> schemas.Category:
+    """Create a new author."""
+    return crud.create_category(category_data=category_data, db=db)
+
+
+@router.get("/categories/", response_model=list[schemas.Category])
+async def read_categories(
+    sort_by: Literal["id", "name"] = "id",
+    sort_order: Literal["asc", "desc"] = "asc",
+    skip: Optional[int] = 0,
```

Comment:
Just for notice, no need to fix: since python 3.10 it's better to use `type | None` syntax instead of `Optional[type]`